### ISSUE TITLE
Calico Manifest needed to support Istio 1.9

### ISSUE THREAD

**Issue by sherwoodzern** [projectcalico/calico#4516]
I'm in the process of setting up a zero trust network for my application. Setting up ZTN requires Istio and I was planning on using Istio 1.9. I see that the Calico Community Edition documentation only mentions the use of 1.6 and 1.7.4. I need to have Calico support the latest release of Istio.

**Comment by axl89** [comment]
I came to the same conclussion after reading the (very well written!) [docs](https://docs.projectcalico.org/security/adopt-zero-trust), which lead me to the [enable application layer policy](https://docs.projectcalico.org/security/app-layer-policy) docs, and everything stopped working when I did the steps metioned in the "Add Calico authorization services to the mesh" section:

```
kubectl apply -f https://docs.projectcalico.org/manifests/alp/istio-app-layer-policy-v1.7.yaml
```

After applying this, the [automatic sidecar injection](https://istio.io/latest/docs/setup/additional-setup/sidecar-injection/#automatic-sidecar-injection) was working well, but I suspect it's not enough..?

**Comment by caseydavenport** [comment]
@mgleung is working on this for Calico v3.20 I believe

**Comment by mgleung** [comment]
This should be fixed in Calico v3.20. The relevant PR is https://github.com/projectcalico/calico/pull/4742
